src/equations/2d/linear_scalar_advection.jl

```diff
@@ -24,18 +24,27 @@ get_name(::LinearScalarAdvectionEquation2D) = "LinearScalarAdvectionEquation2D"
 varnames_cons(::LinearScalarAdvectionEquation2D) = SVector("scalar")
 varnames_prim(::LinearScalarAdvectionEquation2D) = SVector("scalar")
 
+function x_trans_periodic(x)
```

Comment:
You need to explicitly call this `_2d` or otherwise we cannot implement a corresponding method for 3D (and 1D).
```suggestion
# Calculates translated coordinates `x` for a periodic domain
function x_trans_periodic_2d(x)
```

Also, you should add a comment explaining the purpose (feel free to adapt my comment if it is not clear enough)

Reply:
Your comment looks great. I´ll put it in like this.